CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+## 2.0.0
+
+- Replace hash with faster and better finalized hash.
+  This replaces the previous "fxhash" algorithm originating in Firefox
+  with a custom hasher designed and implemented by Orson Peters ([`@orlp`](https://github.com/orlp)).
+  It was measured to have slightly better performance for rustc, has better theoretical properties
+  and also includes a signficantly better string hasher.
+- Fix `no_std` builds
+
+## 1.2.0 (**YANKED**)
```

Comment:
kinda sad that we don't have fxhash version of this crate with the improvements of 1.2.0

Reply:
i don't think it matters much, people who are already using it don't benefit much from the changes, while new users will get 2.0.0

Reply:
Some of the things on the changelog _were_ long awaited, so I'm not sure people who already use it won't benefit from this.